doc/sphinx/using/libraries/funind.rst

```diff
@@ -169,14 +169,21 @@ terminating functions.
 Tactics
 -------
 
-.. tacn:: functional induction (@qualid {+ @term})
+.. tacn:: functional induction @term {? using @one_term {? with @bindings } } {? as @simple_intropattern }
    :name: functional induction
 
-   The tactic functional induction performs case analysis and induction
-   following the definition of a function. It makes use of a principle
+   Performs case analysis and induction
+   following the definition of a function. It uses a principle
    generated by :cmd:`Function` or :cmd:`Functional Scheme`.
    Note that this tactic is only available after a ``Require Import FunInd``.
 
+   .. todo PR (reword or drop)
+      Similar to :tacn:`induction` and :tacn:`elim`, this allows explicitly giving
+      the name of the introduced variables, the induction principle, and
+      the values of dependent premises of the elimination scheme, including
+      *predicates* for mutual induction when :n:`@qualid` is part of a mutually
+      recursive definition.
```

Comment:
The important thing to keep is that:
- the `using` clause is for manually specifying the induction principle / elimination scheme (the two terms mean the same thing);
- the `with` clause is for specifying the arguments of the induction principle;
- the `as` clause is for naming the introduced variables.